### ISSUE TITLE
Ram upgrade

### ISSUE THREAD

**Issue by nerijunior** [leocg/Hackintosh-Dell-G5-5590#30]
Any concerns regarding RAM upgrade? I'll put 2x16gb ddr4 sodimm 2666mhz. Dell website says that's the limit but IDK if I need to change some config for Mac recognize it.

**Comment by leocg** [comment]
I think it should be fine. Switching RAM won't mess with OpenCore setup. You can try removing 1 slot and boot with 8gb first to be sure, but normally memory switch won't affect system stability. I would go for a macOS compatible memory like Crucial or Corsair to be sure. Already used those on real MacBooks without issues. Don't need to be "mac memory", just the brand, If you search on sites like Mercado Livre you will see some products listed as "mac memory", but it's just a normal memory with higher prices. And be sure to use 2x16gb, [this notebook don't support 1x32gb](https://www.dell.com/support/manuals/pt-br/g-series-15-5590-laptop/g5-15-setup-and-specifications/memória?guid=guid-f07064b8-2eb4-49ec-88a8-43b102e42417&lang=pt-br).

I will leave this issue open until you successfully install. Leave a feedback after installation so I can update guide with memory upgrade success. Thanks.

**Comment by nerijunior** [comment]
Sure! Crucial was The cheaper one, I bought two of 16gb, at the end of the week I'll install and post the result.

**Comment by nerijunior** [comment]
Everything working. I installed the new crucial memory today ( the old one was from ADATA ) and everything is running as expected.

![image](https://user-images.githubusercontent.com/17169/216454387-0df3d516-53f2-41bb-89f1-767021a8ba2e.png)
![image](https://user-images.githubusercontent.com/17169/216454560-e871a4ac-c1c3-4237-a582-2cb28941bbc7.png)
![image](https://user-images.githubusercontent.com/17169/216454569-9dcdcae5-a73f-4b47-905c-f781e7d781e9.png)

I didn't run the extensive memory test because it said that will take 24 hours 🤣 

![image](https://user-images.githubusercontent.com/17169/216454667-744744eb-73f3-47fa-9f68-8104365e0e23.png)

If I found any issue I post the update here.

Thanks @leocg
